docs/_docs/04_reference/built-in-functions.md

```diff
@@ -43,7 +43,7 @@ Terragrunt allows you to use built-in functions anywhere in `terragrunt.hcl`, ju
 
 ## OpenTofu/Terraform built-in functions
 
-All [OpenTofu/Terraform built-in functions](https://opentofu.org/docs/language/functions/) are supported in Terragrunt config files:
+All [OpenTofu/Terraform built-in functions (as of v0.15.3)](https://opentofu.org/docs/language/functions/) are supported in Terragrunt config files:
```

Comment:
The combination of OpenTofu and v0.15.3 might be misleading, as OpenTofu v0.15.3 was never officially released.

Reply:
For all intents and purposes, it was, as they share the same lineage, and there's no practical difference to the functions. 

If it bothers you, I'll change it, but I think it's fine.